app/components/works/add_files_component.html.erb

```diff
@@ -24,18 +27,18 @@
           </div>
           <span style="margin: .7rem">or</span> <button type="button" class="dz-clickable btn btn-outline-primary">Choose files</button>
           <div class="invalid-feedback" data-dropzone-target="feedback">You must attach a file<%= error_message %></div>
-          <div class="dropzone-previews dropzone-files-previews" data-dropzone-target="previewsContainer">
-            <%= form.fields_for :attached_files do |file_form| %>
-              <%= render Works::FileRowComponent.new(form: file_form) %>
-            <% end %>
-          </div>
         </fieldset>
```

Comment:
this was the old style of showing a flat list of files ... we will replace this with the new hierarchical components, added to the new `edit` component namespace